libs/utils/tracerAPI.ts

```diff
@@ -43,28 +63,28 @@ export const fetchPoolCommits: (
         to?: number;
         account?: string;
     },
-) => Promise<APICommitReturn[]> = async (network, { pool, from, to, account }) => {
-    const tracerRoute =
-        `${BASE_TRACER_API}/poolsv2/tradeHistory` +
-        `?network=${network}&from=${from ?? 0}&to=${to ?? Math.floor(Date.now() / 1000)}` +
-        `${!!pool ? `&poolAddress=${pool}` : ''}` +
-        `${!!account ? `&userAddress=${account}` : ''}`;
-    const tracerCommits = await fetch(tracerRoute)
+) => Promise<APICommitReturn[]> = async (_network, { pool, account, from }) => {
```

Comment:
is this separate to where we fetch actual trade history? ie its a dupe?

Reply:
yiiiis